ansible/roles/apache/tasks/main.yml

```diff
@@ -13,6 +13,19 @@
     - expires
     - filter
```

Comment:
Needed to add these module updates to the apache2 set up in order to prevent this error:

<img width="929" alt="screenshot 2016-01-30 00 10 41" src="https://cloud.githubusercontent.com/assets/1557348/12697750/d7473c02-c758-11e5-949d-c3416f7ab52f.png">